llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -13,7 +13,9 @@
 #include "llvm/CodeGen/BasicTTIImpl.h"
 #include "llvm/CodeGen/CostTable.h"
 #include "llvm/CodeGen/TargetLowering.h"
+#include "llvm/IR/IRBuilder.h"
```

Comment:
Can this be removed if the unused `IRBuilder` object created in getMemoryRefInfo is removed?